mllib/src/main/scala/org/apache/spark/ml/feature/StopWordsRemover.scala

```diff
@@ -29,14 +29,14 @@ import org.apache.spark.sql.types.{ArrayType, StringType, StructField, StructTyp
 /**
  * stop words list
  */
-private object StopWords {
+protected[spark] object StopWords {
 
   /**
    * Use the same default stopwords list as scikit-learn.
    * The original list can be found from "Glasgow Information Retrieval Group"
    * [[http://ir.dcs.gla.ac.uk/resources/linguistic_utils/stop_words]]
    */
-  val EnglishStopWords = Array( "a", "about", "above", "across", "after", "afterwards", "again",
+  val ENGLISH_STOP_WORDS = Array( "a", "about", "above", "across", "after", "afterwards", "again",
```

Comment:
minor: Since the object is already `StopWords`, would `English` be sufficient? We didn't use `ENGLISH_STOP_WORDS` because it is a mutable array.